full-service/src/service/memo.rs

```diff
@@ -80,6 +83,14 @@ pub trait MemoService {
     ) -> Result<bool, MemoServiceError>;
 }
 
+// validate_sender_memo
+//
+// validating the sender memo includes:
+// 1. Is there a sender memo?
+// 2. Does the provided sender public address hash to the
+//    same value as the memo's sender address hash?
+// 3. When we recreate the HMAC, does it match the
+//    HMAC conveyed in the memo?
```

Comment:
This should be right above the `fn validate_sender_memo` line

Reply:
moved! Thanks for the suggestion.